e164.js

```diff
@@ -5,648 +5,680 @@
-  "1289": [ "CA", "Canada" ],
-  "1301": [ "US", "United States" ],
-  "1302": [ "US", "United States" ],
-  "1303": [ "US", "United States" ],
-  "1304": [ "US", "United States" ],
-  "1305": [ "US", "United States" ],
-  "1306": [ "CA", "Canada" ],
-  "1307": [ "US", "United States" ],
-  "1308": [ "US", "United States" ],
-  "1309": [ "US", "United States" ],
-  "1310": [ "US", "United States" ],
-  "1312": [ "US", "United States" ],
-  "1313": [ "US", "United States" ],
-  "1314": [ "US", "United States" ],
-  "1315": [ "US", "United States" ],
-  "1316": [ "US", "United States" ],
-  "1317": [ "US", "United States" ],
-  "1318": [ "US", "United States" ],
-  "1319": [ "US", "United States" ],
-  "1320": [ "US", "United States" ],
-  "1321": [ "US", "United States" ],
-  "1323": [ "US", "United States" ],
-  "1325": [ "US", "United States" ],
-  "1330": [ "US", "United States" ],
-  "1331": [ "US", "United States" ],
-  "1332": [ "US", "United States" ],
-  "1334": [ "US", "United States" ],
-  "1336": [ "US", "United States" ],
-  "1337": [ "US", "United States" ],
-  "1339": [ "US", "United States" ],
-  "1340": [ "VI", "Virgin Islands, US" ],
-  "1343": [ "CA", "Canada" ],
-  "1345": [ "KY", "Cayman Islands" ],
-  "1346": [ "US", "United States" ],
-  "1347": [ "US", "United States" ],
-  "1351": [ "US", "United States" ],
-  "1352": [ "US", "United States" ],
-  "1360": [ "US", "United States" ],
-  "1361": [ "US", "United States" ],
-  "1364": [ "US", "United States" ],
-  "1365": [ "CA", "Canada" ],
-  "1367": [ "CA", "Canada" ],
-  "1380": [ "US", "United States" ],
-  "1385": [ "US", "United States" ],
-  "1386": [ "US", "United States" ],
-  "1401": [ "US", "United States" ],
-  "1402": [ "US", "United States" ],
-  "1403": [ "CA", "Canada" ],
-  "1404": [ "US", "United States" ],
-  "1405": [ "US", "United States" ],
-  "1406": [ "US", "United States" ],
-  "1407": [ "US", "United States" ],
-  "1408": [ "US", "United States" ],
-  "1409": [ "US", "United States" ],
-  "1410": [ "US", "United States" ],
-  "1412": [ "US", "United States" ],
-  "1413": [ "US", "United States" ],
-  "1414": [ "US", "United States" ],
-  "1415": [ "US", "United States" ],
-  "1416": [ "CA", "Canada" ],
-  "1417": [ "US", "United States" ],
-  "1418": [ "CA", "Canada" ],
-  "1419": [ "US", "United States" ],
-  "1423": [ "US", "United States" ],
-  "1424": [ "US", "United States" ],
-  "1425": [ "US", "United States" ],
-  "1430": [ "US", "United States" ],
-  "1431": [ "CA", "Canada" ],
-  "1432": [ "US", "United States" ],
-  "1434": [ "US", "United States" ],
-  "1435": [ "US", "United States" ],
-  "1437": [ "CA", "Canada" ],
-  "1438": [ "CA", "Canada" ],
-  "1440": [ "US", "United States" ],
-  "1441": [ "BM", "Bermuda" ],
-  "1442": [ "US", "United States" ],
-  "1443": [ "US", "United States" ],
-  "1445": [ "US", "United States" ],
-  "1450": [ "CA", "Canada" ],
-  "1456": [ "US", "United States" ],
```

Comment:
There are a lot of ones removed, like this one. Is that an issue? Will it cause errors?

Reply:
I specifically commented about this in the tester file.  This number is actually Canadian.  And was replaced properly in the new dataset.